configure.ac

```diff
@@ -672,7 +672,6 @@ AC_SUBST(ARCH_CCASFLAGS)
 AC_CONFIG_FILES(Makefile \
   docs/version.entities \
   docs/help/bt-edit/Makefile \
-  docs/help/bt-edit/C/Makefile \
```

Comment:
I've folded the tasks in "bt-edit/C/Makefile" into "bt-edit/Makefile" to deal with out-of-tree build issue I had. There's a comment in "bt-edit/Makefile" explaining why.